website/docs/sdks/python.md

```diff
@@ -29,7 +29,10 @@ client.is_enabled("My Toggle")
 Specifying a default value:
 
 ```Python
-client.is_enabled("My Toggle", default_value=True)
+def custom_fallback(feature_name: str, context: dict) -> bool:
+    return True
+         
+client.is_enabled("My Toggle", fallback_function=custom_fallback)
```

Comment:
This isn't wrong but a lambda would point out that this is different from the previous call, as per @thomasheartman's comments and also be a bunch simpler which is always nice. Something like 

```unleash_client.is_enabled("ThisFlagDoesn'tExist", fallback_function=lambda: True)```

What do you think?